desktop_version/src/FileSystemUtils.cpp

```diff
@@ -213,10 +213,11 @@ static bool FILESYSTEM_exists(const char *fname)
 	return PHYSFS_exists(fname);
 }
 
-void FILESYSTEM_mount(const char *fname)
+#define ASSETS_MOUNT_PATH ".vvv-mnt/assets/"
```

Comment:
Everything makes sense to me except for this name. At first I panicked because it looked like we were making a new filesystem directory, but it turns out this is just a virtual path... and for a very specific type of asset. This needs to be renamed to something screamingly obvious as to what exactly it does, so that if someone reads literally zero lines of code except for this one they know _exactly_ why it exists with absolutely zero context. If we can ever get this design issue fixed in PhysFS this line may be a big help in explaining what the issue is.

Reply:
Renamed to `VIRTUAL_MOUNT_PATH`. Is that okay enough?

Reply:
Still have no idea what ".vvv-mnt/assets/" means. Is it where all files go? The original files? Is that why vvv is there? Or is it for custom files? In either case, why the dot? Is this a hidden folder on Unix systems?

Reply:
Ok. How about `.vvv-mnt/custom-assets/`? I put in a dot because it's a virtual path, to lessen the chances that there would actually be an existing directory there (it's what Ved does with `.ved-sys`).

Reply:
`.vvv-mnt-virtual-%s/custom-assets/` where %s is a random string will probably yield the best result - it's a virtual directory, is nondeterministic (i.e. "do not hardcode to this location"), and it's for custom assets only. That I think will cover it!

Reply:
Alright, done.